src/index.js

```diff
@@ -508,6 +508,17 @@ Stream.prototype.skipWhile = function (p) {
   return skipWhile(p, this)
 }
 
+/**
+ * stream:                        -123456789->
+ * skipAfter(x => x === 5, stream): ---12345|
```

Comment:
This is also showing a delay:

```js
/**
 * stream:                        -123456789->
 * skipAfter(x => x === 5, stream): ---12345|
```

Let's remove the delay and make sure the events are aligned:

```js
/**
 * stream:                          -123456789->
 * skipAfter(x => x === 5, stream): -12345|
```